coverxygen/__init__.py

```diff
@@ -33,9 +33,9 @@ def __init__(self, p_path, p_output, p_scope, p_kind, p_prefix, p_format, p_root
     self.m_output  = p_output
     self.m_scope   = p_scope
     self.m_kind    = p_kind
-    self.m_prefix  = p_prefix
+    self.m_prefix  = os.path.abspath(p_prefix) if p_prefix is not None else ""
     self.m_format  = p_format
-    self.m_rootDir = p_rootDir
+    self.m_rootDir = os.path.abspath(p_rootDir) if p_rootDir is not None else ""
```

Comment:
I realized that it's a bad idea to convert the paths to absolute paths here.
I'll figure out something better.

Reply:
Ah... already merged.
I'll create another PR with a fix then.

Reply:
Nevermind. I thought that coverxygen produced relative paths before but it always produced absolute paths so the change is fine.